src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -110,6 +115,8 @@ private IHttpActionResult AddOrUpdateFunctionSecret(string keyName, string value
                 operationResult = _secretManager.AddOrUpdateFunctionSecret(keyName, value, functionName);
             }
 
+            _traceWriter.VerboseFormat(Resources.TraceKeysApiSecretChange, keyName, functionName ?? "host", operationResult.Result);
```

Comment:
When working with resource strings, string interpolation isn't an option, so this just reduces the verbosity over having to call `string.Format` on those tracing calls.